docker/k8s/pmm-client/Dockerfile

```diff
@@ -1,35 +0,0 @@
-# Copyright 2019 The Vitess Authors.
```

Comment:
This was added originally for Helm templates, but those Helm templates have been long since deprecated and removed, so we can also remove this.

Reply:
last pushed to docker hub 2 years ago.